### PR TITLE
Update readme to comply with trademark rules

### PR DESCRIPTION
**Pull request by jlprat**
Add disclaimer for trademark.
Add full name to Apache Kafka in places where it made sense.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -555,6 +555,15 @@ Recent contributors are listed on the project's GitHub `contributors page`_.
 
 .. _`contributors page`: https://github.com/aiven/journalpump/graphs/contributors
 
+Trademark
+=========
+
+Apache Kafa is either registered trademark or trademark of the Apache Software
```
Comment (m-novikov) [inline README.rst:561]: Typo `Apache Kafa`

File: README.rst [hunk 2]
```diff
@@ -555,6 +555,15 @@ Recent contributors are listed on the project's GitHub `contributors page`_.
 
 .. _`contributors page`: https://github.com/aiven/journalpump/graphs/contributors
 
+Trademark
+=========
+
+Apache Kafa is either registered trademark or trademark of the Apache Software 
+Foundation in the United States and/or other countries. Elasticsearch, 
+AWS CloudWatch,logplex and rsyslog are trademarks and property of their respective
```
Comment (m-novikov) [inline README.rst:563]: missing space after `,` in `,logplex`

**Review by m-novikov — APPROVED** [review]

**Review by m-novikov — CHANGES_REQUESTED** [review]
LGTM :) could you squash commits

**Comment by jlprat** [comment]
Squashed

**Review by m-novikov — APPROVED** [review]

**Pull request merged**